lib/src/rules/no_boolean_literal_compare_rule.dart

```diff
@@ -10,6 +10,12 @@ import 'rule_utils.dart';
 // Inspired by TSLint (https://palantir.github.io/tslint/rules/no-boolean-literal-compare/)
 
 class NoBooleanLiteralCompareRule extends BaseRule {
+  static const _failureComprareNullAwarePropertyWithTrue =
+      "Compare boolean null aware property with boolean literal 'true' is not semantic recommended.";
```

Comment:
`Comparison of null-conditional boolean with boolean literal may result in comparing null with boolean.`